bin/pyenv-offline-installer

```diff
@@ -100,4 +100,11 @@ if ! command -v pyenv 1>/dev/null; then
       ;;
     esac
   } >&2
+elif [ -d "${PYENV_ROOT}" ]; then
+  { echo
+    colorize 1 "WARNING"
+    echo ": Kindly remove '.pyenv' from ${HOME} to proceed with installation."
```

Comment:
same here.

Reply:
@obestwalter I have made the necessary changes.